### PR TITLE
Documentation for New PowerToys Toy (Paste As Plain Text)

### PR DESCRIPTION
**Pull request by ethanfangg**
Pending release of PowerToys v0.68.0 (should land in March):
- added paste-as-plain-text.md
- updated index.md
- added png

### CODE REVIEW

**Review by jaimecbernardo — COMMENTED** [review]
Hi @ethan-fang-MS ,
There should be a mention that Paste as plain text replaces the current clipboard entry with the unformatted text.

File: hub/powertoys/paste-as-plain-text.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+description: Allows you to paste text from your clipboard, excluding text-formatting, with a quick key shortcut.
+ms.date: 02/10/2023
+ms.topic: article
+ms.localizationpriority: medium
+no-loc: [PowerToys, Windows, PowerToys Paste as Plain Text, Paste as Plain Text, Win]
+---
+
+# Paste as Plain Text
+
+Allows you to paste text from your clipboard, excluding text-formatting, with a quick key shortcut.
```
Comment (crutkas) [inline hub/powertoys/paste-as-plain-text.md]: I feel we should directly comment it will replace your clipboard with the unformatted version.

Comment (mattwojo) [reply hub/powertoys/paste-as-plain-text.md]:
```suggestion
PowerToys **Paste as plain text** enables you to paste text stored in your clipboard, excluding any text-formatting, using a quick key shortcut. Any formatting included with the clipboard text will be replaced with an unformatted version of the text.
```

File: hub/powertoys/paste-as-plain-text.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+![Paste as Plain Text screenshot.](../images/pt-paste-as-plain-text.png)
+
+## Getting started
+
+### Enable
+
+To start using Paste as Plain Text, enable it in the PowerToys Settings (Paste as Plain Text section).
```
Comment (mattwojo) [inline hub/powertoys/paste-as-plain-text.md:20]: **Paste as Plain Text** <- suggestion to bold mentions of the tool title

**Comment by Jay-o-Way** [comment]
@mattwojo v0.68 was released like 10 minutes ago 🙂

**Review by mattwojo — APPROVED** [review]

**Pull request merged**